src/type/type.js

```diff
@@ -162,10 +162,15 @@ function getTypes(graffitiModels) {
   return types;
 }
 
+function addType(name, type) {
```

Comment:
You need to define a function before using it to make eslint happy (`npm run eslint`).